queries/c/highlights.scm

```diff
@@ -177,7 +180,11 @@
   declarator: (identifier) @parameter)
 
 (parameter_declaration
-  declarator: (pointer_declarator) @parameter)
+  declarator: (pointer_declarator) @pointer.declarator (#set! "priority" 101))
```

Comment:
make the c pointer declarator `*` (star) character a new hl group to make it possible to color it (blue in example image)
Without priority the `operator` hl group would override it.

Reply:
there's a separate PR to highlight it as `@type`   #3874 . If you're introducing something new it should be a sub-scope of an existing category (e.g. `@operator.pointerdeclaration`)